todo_app/app.py

```diff
@@ -1,14 +1,20 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['POST', 'GET'])
```

Comment:
It's not obvious what POST to '/' means.

Would be nicer to have a descriptive path like '/add-item'